src/components/Form/Form.tsx

```diff
@@ -0,0 +1,161 @@
+/* eslint-disable */
```

Comment:
you should not disable eslint! It is very important that we lint our files so we can ensure high, consistent quality of our code
```suggestion
```